src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static Boolean isRunning = true;
+
+    public static void printGreetings() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+    }
+
+    public static String findTaskSpecifics(String details) {
+        return details.substring(details.indexOf(" "), details.indexOf("/"));
```

Comment:
1. Avoid complicated expressions. Perhaps consider splitting the logic into mutiple lines and/or methods to make the code easier to read. 
2. Avoid using magic strings. Consider adding the strings into a descriptive variable.